packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -55,6 +58,19 @@ class BaseDescopeWc extends HTMLElement {
 
   #init = false;
 
+  loggerWrapper = {
+    error: (message: string, description = '') => {
+      this.logger.error(message, description, new Error());
```

Comment:
Now when we are getting the logger, some fns might not be implemented
I suggest to trigger it only if it exists, e.g.
```
this.logger.error?.(...)
```